### PR TITLE
Guarin lig 2221 add tag info to runs pip

### PR DESCRIPTION
**Pull request by guarin**
## Changes

* Add `get_compute_worker_run_tags` endpoint

## How was it tested?

* Added unit tests
* Manual tests

### CODE REVIEW

File: tests/api_workflow/test_api_workflow_compute_worker.py [hunk 1]
```diff
@@ -557,6 +554,82 @@ def test_download_compute_worker_run_artifacts(mocker: MockerFixture) -> None:
...
+        name="tag-0",
+        tot_size=0,
+        created_at=0,
+        changes=dict(),
+        run_id="run-0",
+    )
+    tag_1 = TagData(
+        id="tag-1",
+        dataset_id="dataset-0",
+        prev_tag_id=None,
```
Comment (japrescott) [inline tests/api_workflow/test_api_workflow_compute_worker.py:604]: technically should be the id of tag_0

File: tests/api_workflow/test_api_workflow_compute_worker.py [hunk 2]
```diff
@@ -557,6 +554,82 @@ def test_download_compute_worker_run_artifacts(mocker: MockerFixture) -> None:
...
+        tot_size=0,
+        created_at=1,
+        changes=dict(),
+        run_id="run-0",
+    )
+    # tag from a different dataset
+    tag_2 = TagData(
+        id="tag-2",
+        dataset_id="dataset-1",
+        prev_tag_id=None,
```
Comment (japrescott) [inline tests/api_workflow/test_api_workflow_compute_worker.py:616]: id of tag_0 or tag_1

**Review by japrescott — APPROVED** [review]

File: tests/api_workflow/test_api_workflow_compute_worker.py (hunk 1 above)
Comment (guarin) [reply tests/api_workflow/test_api_workflow_compute_worker.py]: Ah yes, thanks!

File: tests/api_workflow/test_api_workflow_compute_worker.py (hunk 2 above)
Comment (guarin) [reply tests/api_workflow/test_api_workflow_compute_worker.py:616]: I think this one is correct because the `dataset_id` is different.

Comment (japrescott) [reply tests/api_workflow/test_api_workflow_compute_worker.py:616]: ah true sorry, didn't see that

**Pull request merged**